gulp/tasks/clean.js

```diff
@@ -4,6 +4,18 @@ const configClean = require( '../config' ).clean;
 const del = require( 'del' );
 const gulp = require( 'gulp' );
 
+
+gulp.task( 'clean:css', () => {
+  // Clean CSS out of /cfgov/static_built/css/
+  del( configClean.css + '/**/*' );
+} );
+
+gulp.task( 'clean:js', () => {
+  // Clean JavaScript out of /cfgov/static_built/js/
+  del( configClean.js + '/**/*' );
+} );
```

Comment:
@sebworks I addressed your earlier comment by getting over my fear of anonymous arrow functions and having the format of this match the original `clean` task below. Let me know if that suits you.